### PR TITLE
move copy-to-latest-channel to its own stage

### PR DESCRIPTION
**Pull request by riarenas**
Fixes the gap outlined in: https://github.com/dotnet/arcade/issues/3982

Test build incoming once the fix for the arcade templates makes it to the latest channel but want to get early feedback on the approach.

### CODE REVIEW

**Comment by riarenas** [comment]
We will also need this change in release/3.x, so once I have validated the fix I will create an ask mode template for it.

**Comment by riarenas** [comment]
Test build https://dev.azure.com/dnceng/internal/_build/results?buildId=360262&view=results looks how I expect it to

File: azure-pipelines.yml [hunk 1]
```diff
@@ -210,3 +193,28 @@ stages:
         -TsaRepositoryName "Arcade-Validation"
         -TsaCodebaseName "Arcade-Validation"
         -TsaPublish $True'
+
+  - stage: Push_to_latest_channel
```
Comment (chcosta) [inline azure-pipelines.yml]: maybe specify a display name?

Comment (riarenas) [reply azure-pipelines.yml]: Yep, that was in my to do list. Thanks for the reminder

**Pull request merged**